concordium-std/Cargo.toml

```diff
@@ -20,11 +20,11 @@ ed25519-zebra = { version = "2.2", optional = true }
 
 [dependencies.concordium-std-derive]
 path = "../concordium-std-derive"
-version = "4"
+version = "4.1"
```

Comment:
I think this will need to be 4.2. 4.1 is already released, and does not support the `event` option (There was a small release of this last week adding a tiny feature).